theme/static/js/hs_file_browser.js

```diff
@@ -1999,6 +2021,11 @@ $(document).ready(function () {
         });
     });
 
+    // Preview data
+    $("[data-fb-action='preview']").click(function () {
+        previewData();
+    });
+
```

Comment:
@kjlippold I don't seem to see any place where uiActionStates.preview.disabled is set to False and don't seem to be able to look at preview on the deployed test server, so I am wondering whether preview is working or not.

Reply:
There's a GeoServer instance linked to prod and beta, but I haven't had a chance to set one up for dev-hs-6. By default, the links use the prod GeoServer, but since the resources on dev-hs-6 aren't registered on that GeoServer, you would expect to see a 404 page generated by Apache when you click the preview link. I'm working on getting another GeoServer set up, and I'll let you know once it's ready.